code/modules/mob/living/carbon/human/say.dm

```diff
@@ -91,6 +91,9 @@
 	var/obj/item/organ/internal/L = get_organ_slot("lungs")
 	if((breathes && !L) || breathes && L && (L.status & ORGAN_DEAD))
 		return FALSE
+	if(oxyloss > 10 || losebreath >= 4)

```

Comment:
Will this silence people low on blood?

Reply:
Yeahp.

Reply:
How low does blood have to be for this to occur? Could pose an issue if it happens very early into blood loss.

Reply:
Once blood loss incurs oxyloss over 10, then you'll be silenced.

You're going to have to be fairly badly hurt for this to occur; you'll have to lose over 125 units of blood before you have to worry about O2 outpacing your ability to breathe; even then, it's going to accumulate very slowly.